tests/fixtures/app/nested-project/actual-project/package.json

```diff
@@ -56,6 +58,10 @@
     "eslint-plugin-qunit": "^7.0.0",
     "loader.js": "^4.7.0",
     "prettier": "^2.2.1",
+    "stylelint": "^14.16.1",
```

Comment:
Should this be alphabetized to be below `qunit-dom`?

Reply:
Good point, fixed.